bin/prepare-gcloud

```diff
@@ -1,27 +1,19 @@
 #!/bin/bash -e
 echo "Preparing the gcloud command line tools."
 
-if [[ -z ${GCP_PROJECT+x}  ]]; then
-  echo "Missing GCP_PROJECT environment variable.  Aborting"
+if [[ -z ${GCP_PROJECT+x} ]] || [[ -z ${GCP_ZONE} ]] || [[ -z ${GCLOUD_KEY+x} ]] || [[ -z ${CLUSTER_NAME+x} ]]; then
+  echo "At least 1 required variable is not set [GCP_PROJECT, GCP_ZONE, GCLOUD_KEY, CLUSTER_NAME]"
```

Comment:
Is the ambiguity in which variable is not set part of the feature for explaining all the vars that are needed? In which case it would stop the "add var, test, see it breaks" loop? Other files have each var checked and error without presenting all the vars needed.